types/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ All notable changes to this project will be documented in this file.  The format
 ### Changed
 * Disable checksummed-hex encoding, but leave checksummed-hex decoding in place.
 * Increase `DICTIONARY_ITEM_KEY_MAX_LENGTH` to 128.
+* Fixed some integer casts.
```

Comment:
We need to mention the new error variant and communicate that to the downstream clients.

Reply:
Seems like this wasn't actioned?

Reply:
Added an entry describing newly added variant